src/TraceEvent/Computers/TraceManagedProcess.cs

```diff
@@ -567,6 +567,30 @@ internal static void SetupCallbacks(TraceEventDispatcher source)
                             }
                         }
                     }
+
+                    if (data.Reason == GCSuspendEEReason.SuspendForGC)
+                    {
+                        TraceGC gc = new TraceGC(mang.GC.m_stats.HeapCount);
+                        gc.Index = mang.GC.GCs.Count;
+                        mang.GC.GCs.Add(gc);
```

Comment:
> mang.GC.GCs.Add(gc); [](start = 23, length = 21)

this looks like the only place you are adding to the GCs list. we can have a GC that happens without the SuspendEEStart event because it's the GC do we before we do the BGC, how do you handle that?